sklearn/model_selection/_search.py

```diff
@@ -1,6 +1,8 @@
 """
 The :mod:`sklearn.model_selection._search` includes utilities to fine-tune the
 parameters of an estimator.
+
+.. versionadded:: 0.18
```

Comment:
I'm unsure of the history of the items in this file -- should I give each class/function in this file the versionchanged directive you suggest above?